opba-tppbanking-services/src/test/java/de/adorsys/opba/tppbanking/services/ais/GetAccountsServiceMockImplTest.java

```diff
@@ -0,0 +1,41 @@
+package de.adorsys.opba.tppbanking.services.ais;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(classes = GetAccountsServiceMockImplTest.TestConfiguration.class)
+class GetAccountsServiceMockImplTest {
+    @Autowired
+    private GetAccountsService getAccountsServiceMock;
+
+    @Test
+    void getAccountsFor() {
+        Optional<AccountsReport> accounts = getAccountsServiceMock.getAccountsFor(new PsuConsentSession());
+        assertTrue(accounts.isPresent());
```

Comment:
We should standartise what kind of assertions are used in project 
org.junit.jupiter.api.Assertions
or
AssertJ

Reply:
Why? :)
They both are available for us.

Reply:
For code style - having consistent assertion usage improves readability